app/components/contact-page-components/ContactDetails.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import Image from "next/image";
+import { useTheme } from "next-themes";
+import { Tooltip } from "@nextui-org/react";
+
+import { socialMediaButtonsContactPage } from "@/constants";
+import {
+  call,
+  callWhite,
+  message,
+  messageWhite,
+} from "@/public/svg-icons/tech-icons";
+
+const ContactDetails = () => {
+  const { theme } = useTheme();
+
+  return (
+    <>
+      <div className="mt-24 flex w-full flex-col md:w-2/5 lg:mt-0">
+        <p className="text-lg font-light text-black300 dark:text-white900 md:text-2xl">
+          My Socials
+        </p>
+        <div className="mt-3.5 flex gap-9 md:mt-7">
+          {socialMediaButtonsContactPage.map((button) => (
+            <Tooltip key={button.title} content={button.title}>
+              <Image
+                src={theme === "light" ? button.icon : button.iconDark}
+                alt="Image of social media icons"
```

Comment:
For the social media icons, each `alt` is set to `"Image of social media icons".` This is the same for every social media button, which doesn't make it very descriptive. It would be better to have more descriptive alt tags, especially for the social media icons. Given that these are being mapped from an array, try using the `button.title` to make each `alt` tag unique and descriptive. 

I mentioned this in the heading already.

Reply:
done and done